.github/workflows/auto-merge/automerge

```diff
@@ -47,7 +47,7 @@ AUTH_HEADERS = {
 def create():
     url = f'{API_URL}/repos/{OWNER}/{REPO_NAME}/pulls'
     params = {
-        'title': f'[auto-merge] {HEAD} to {BASE}',
+        'title': f'[auto-merge] {HEAD} to {BASE} [ci skip]',
```

Comment:
Small request -  can this be [skip ci] to be consistent with the rapids.ai repo?  
```suggestion
        'title': f'[auto-merge] {HEAD} to {BASE} [skip ci]',
```